inst/WORDLIST

```diff
@@ -1,9 +1,32 @@
+args
```

Comment:
@vedhav can you tick out `args` in your documentation so that we do not need to add this to WORDLIST?